cfme/tests/configure/test_multi_tenancy_manual.py

```diff
@@ -0,0 +1,438 @@
+# -*- coding: utf-8 -*-
+# pylint: skip-file
```

Comment:
Why is this included here? We want to lint all python

Reply:
I tried to keep the same syntax as set in test_manual.py, 
Thank you for the suggestion, I have removed it now.